src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -15,13 +15,16 @@
 #include "ray/raylet/scheduling/cluster_resource_scheduler.h"
 
 #include <boost/algorithm/string.hpp>
+#include <memory>
+#include <string>
+#include <vector>
 
 #include "ray/common/grpc_util.h"
 #include "ray/common/ray_config.h"
 
 namespace ray {
 
-using namespace ::ray::raylet_scheduling_policy;
+using namespace ::ray::raylet_scheduling_policy; // NOLINT
```

Comment:
nit: I would prefer to avoid `using namespace` here, but not a big issue, no need to change.